src/stylesheets/components/_list-styles.scss

```diff
@@ -82,10 +82,12 @@ dl, dt, dd {
 }
 .list--inline {
   @include list--inline;
-}
-
-article footer .list--inline {
-  margin-top: round($base-spacing-unit * 9);
+  a, span {
+    display: inline;
+    padding: 0;
```

Comment:
Same here, choose default values for the mixin or class `list--inline`. Any other values for margin, padding and display types components that include `list--inline` need, should be on the component class (i.e navigation--letter or navigation--number)